network-api/networkapi/templates/fragments/buyersguide/pni_newsletter_box.html

```diff
@@ -11,9 +11,8 @@
         class="join-us tw-bg-none"
         data-form-position="callout-box"
         data-form-style="pni"
-        data-cta-header="{% trans "Help shape the future of the web for the public good." %}"
-        data-cta-description="<p class='tw-mb-12'>{% trans "Join our Mozilla News email list to get action alerts & internet tips right to your inbox." %}</p>"
+        data-cta-header="{% trans cta_header|default:"Help shape the future of the web for the public good." %}"
```

Comment:
Where do we get `cta_header` and `cta_description` from?

Reply:
from the template that `{% include` this fragments

Reply:
I see. In this case, I think this template construction became a bit weird with the default parameters. Maybe this will be a better syntax: 

https://stackoverflow.com/questions/3595099/django-trans-tag-within-a-default-filter

```
data-cta-header="{{ cta_header|default:_("Help shape the future of the web for the public good.") }}"
data-cta-description="<p class='tw-mb-12'>{{ cta_description|default:_("Join our Mozilla News email list to get action alerts & internet tips right to your inbox.") }}</p>"
```